gdnative-core/src/core_types/string.rs

```diff
@@ -260,6 +260,36 @@ impl std::hash::Hash for GodotString {
     }
 }
 
+impl Add<&GodotString> for &GodotString {
```

Comment:
Suggestion: What about also adding a `Add<&str> for &GodotString`? That way doing `some_gd_string + "test"` will work too.

Reply:
Added one for `AsRef<str>`

Reply:
Maybe also one for `AddAssign`?